src/components/CreateEventForm/CreateEventForm.jsx

```diff
@@ -24,137 +24,122 @@ function CreateEventForm({createEventOpen, onCreateEventClick, activeEvent, onCh
 
     const handleChange = (event) => {
         // if (auth.token){
```

Comment:
maybe remove the unused commented out code

Reply:
Done :D